src/core/screendimmer.cpp

```diff
@@ -25,14 +25,22 @@ ScreenDimmer::ScreenDimmer( QgsApplication *app )
   connect( app, &QGuiApplication::applicationStateChanged, this, [=]() { setSuspend( app->applicationState() != Qt::ApplicationActive ); } );
 
   mTimer.setSingleShot( true );
-  mTimer.setInterval( 20000 );
+  mTimer.setInterval( mTimeoutS * 1000 );
```

Comment:
Can you remove the S suffix here? Or spell it out properly (i.e. mTimeoutSeconds). I personally think the S is not needed.

Reply:
I always like to have the unit of measure in the var nme. Is `timeoutSecs` ok with you, or you prefer the long version?

Reply:
@suricactus , I could live with Secs :wink: